netcore/Makefile

```diff
@@ -126,7 +126,7 @@ update-tests-corefx: corefx/.stamp-dl-corefx-tests-$(NETCORETESTS_VERSION)
 
 corefx/.stamp-dl-corefx-tests-$(NETCORETESTS_VERSION):
 	curl $(TEST_ASSETS_URL) --output corefx-test-assets.xml
-	python dl-test-assets.py corefx-test-assets.xml $(FEED_BASE_URL) corefx/tests
```

Comment:
Unrelated changes ?

Reply:
@vargaz on our CI `python` stands for Python 2 (default for macOS) and https://github.com/mono/mono/pull/15524 made that script python3 only

Reply:
I mean the change is unrelated to the PR.

Reply:
I understand but without it I will not be able to see CI results for this PR 🙂 all netcore lanes are currently crash on the `download corefx tests` step in master. I can file a separate PR for it.